src/renderer/webpack.dev.js

```diff
@@ -1,32 +0,0 @@
-const path = require("path");
-const webpack = require("webpack")
-const WebpackNotifierPlugin = require("webpack-notifier")
-const merge = require('webpack-merge');
-const common = require('./webpack.common.js');
-const FlowBabelWebpackPlugin = require('./flowtype-plugin');
```

Comment:
is flow working with this webpack config?

Reply:
Now it does.